pkg/quarterdeck/db/models/users.go

```diff
@@ -344,6 +360,107 @@ func (u *User) UpdateLastLogin(ctx context.Context) (err error) {
 	return tx.Commit()
 }
 
+const (
+	userInviteSQL = "INSERT INTO user_invitations (organization_id, email, expires, token, secret, created_by, created, modified) VALUES (:orgID, :email, :expires, :token, :secret, :createdBy, :created, :modified)"
+)
+
+// Create an invitation in the database from the user to the specified email address
+// and return the invitation token to send to the user.
+func (u *User) Invite(ctx context.Context, email string) (token string, err error) {
+	var (
+		invite *db.VerificationToken
+	)
+
+	// Create a token that expires in 7 days
+	if invite, err = db.NewVerificationToken(email); err != nil {
```

Comment:
I'm not sure yet if we need to include the orgID in the token or we will just have Quarterdeck do the database lookup.

Reply:
Either is fine with me.